cli/devicecontroller.js

```diff
@@ -50,8 +50,10 @@ function storeSdkServerConfiguration(brain, sdkOptions, devices) {
   const { serverPort, serverName } = sdkOptions;
   serverConfiguration = {
     brain,
-    port: serverPort || 6336,
+    port: serverPort || process.env.NEEO_SERVER_PORT || 6336,
     name: serverName || 'default',
+    adapterIpAddress: process.env.NEEO_SERVER_IP,
+    baseurl: process.env.NEEO_SERVER_BASEURL,
```

Comment:
Can you explain why the baseurl variable is needed here? it doesn't seem used below and the url is generated from the IP address.

Reply:
in https://github.com/NEEOInc/neeo-sdk/blob/master/lib/device/index.js#L48 its only generated when `conf.baseurl` is not set. With this change, you can set `conf.baseurl` with an ENV 

Generating the baseURL by IP Address might not always work (e.g. behind NAT), I even would suggest to not build at all a baseURL that needs to be send to the Brain. But thats an other Topic and probably a bit more tricky.

Reply:
oh OK makes sense, I didn't see it in the PR and I was confused, my bad.

Reply:
it's not in the PR :) it was already there. np :+1: